src/app/services/explorer/explorer.service.ts

```diff
@@ -54,24 +57,34 @@ export class ExplorerService {
     return this.internalMaxDecimals;
   }
 
-  /**
-   * Lets know if the initialize() function has already been called.
-   */
-  private initialized = false;
+  private nodeUrlSubscription: Subscription;
+  private initializationSubscription: Subscription;
 
   constructor(
     private api: ApiService,
   ) { }
 
+  /**
+   * Initializes the service.
+   */
   initialize() {
-    if (this.initialized) {
-      return;
-    }
-    this.initialized = true;
+    namedAddresses.forEach(namedAddress => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
    namedAddresses.forEach((namedAddress) => {
```